src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -900,6 +900,14 @@ export default (accessToken, userID, opts) => {
       {},
       { headers: true }
     ),
+    partnerContactLoader: gravityLoader<
+      any,
+      { partnerId: string; contactId: string }
+    >(
+      ({ partnerId, contactId }) => `partner/${partnerId}/contact/${contactId}`,
+      {},
+      { headers: true }
```

Comment:
Probably best to only return the headers form when you need it (like for connection pagination)

Reply:
Hmm, I wonder if there is some auth thing going on, when I remove the headers I start getting `null` back for contact, will dig a bit

Reply:
The signature of the data that is resolved from the loader changes when you remove `headers: true`, there's no more `{body, headers}`, it's just the actual data (so `const contact = await ....`).

Reply:
:doh: yes that would do it